rockpaperscissors/src/modules/Game.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import { useState } from "react";
+import styles from './Game.module.css';
+
+const CHOICES = [
+    { name: "rock", emoji: "✊" },
+    { name: "paper", emoji: "✋" },
+    { name: "scissors", emoji: "✌️" }
+];
+
+const Game = () => {
+    const [playerChoice, setPlayerChoice] = useState(null);
+    const [compChoice, setCompChoice] = useState(null);
+    const [result, setResult] = useState(null);
+
+    const handlePlayerChoice = (choice) => {
+        const compChoice = CHOICES[Math.floor(Math.random() * CHOICES.length)];
+        setPlayerChoice(choice);
+        setCompChoice(compChoice);
+
+        if (choice.name === compChoice.name) {
+            setResult("Tie!");
+          } else if (
+            (choice.name === "rock" && compChoice.name === "scissors") ||
+            (choice.name === "paper" && compChoice.name === "rock") ||
+            (choice.name === "scissors" && compChoice.name === "paper")
+          ) {
+            setResult("You win!");
+          } else {
+            setResult("You lose!");
+          };
+    };
+
+    const resetGame = () => {
+        setPlayerChoice(null);
+        setCompChoice(null);
+        setResult(null);
+    };
+
+    return (
```

Comment:
you can try to separate component to presentation (only display informations) and logic ones (keep state and methods to update the state).